### PR TITLE
Hackathon: Tictactoe MoveApplier

### PR DESCRIPTION
**Pull request by Cypher1**

### CODE REVIEW

File: particles/TicTacToe/MoveApplierTest.recipe [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  create as h1
+  create as h2
+  create as h3
+  create as h4
+  MoveApplier
+    board = h0
+    nextMove = h1
+    state = h2
+    boardResult = h3
+    messageResult = h4
```
Comment (shans) [inline particles/TicTacToe/MoveApplierTest.recipe]: arrow these the right way.

File: particles/TicTacToe/particles/MoveApplier.particle [hunk 2]
```diff
@@ -0,0 +1,10 @@
+schema MoveResult // Updated board or error message.
+  (Number, Number, Number, Number, Number, Number, Number, Number, Number) result
+  Text error
```
Comment (shans) [inline particles/TicTacToe/particles/MoveApplier.particle]: we don't need this now

File: particles/canonical.manifest [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-import 'TVMaze/TVMaze.recipes'
-import 'Words/Words.recipes'
-import 'Demo/Demo.recipes'
+// import 'Products/Products.recipe'
+// import 'Events/Events.recipes'
+// import 'Restaurants/Restaurants.recipes'
+// import 'Music/Music.recipes'
+// import 'TVMaze/TVMaze.recipes'
+// import 'Words/Words.recipes'
+// import 'Demo/Demo.recipes'
```
Comment (shans) [inline particles/canonical.manifest]: uncomment

Comment (Cypher1) [reply particles/canonical.manifest]: Done

**Pull request merged**